als-common/shared/src/main/scala/org/mulesoft/amfintegration/amfconfiguration/AmfConfigurationWrapper.scala

```diff
@@ -32,21 +36,31 @@ import scala.concurrent.Future
   * @param alsDialectProvider Dialects initialized with the ALS Server
   * @param platform
   */
-class AmfConfigurationWrapper private[amfintegration] (private val initialConfig: AMFConfiguration,
-                                                       val alsVocabularyRegistry: AlsVocabularyRegistry,
-                                                       val alsDialectProvider: BaseAlsDialectProvider,
-                                                       val resourceLoaders: Seq[ResourceLoader])
+class AmfConfigurationWrapper private[amfintegration] (
+    private val initialConfig: AMFConfiguration,
+    val alsVocabularyRegistry: AlsVocabularyRegistry,
+    val alsDialectProvider: BaseAlsDialectProvider,
+    val resourceLoaders: Seq[ResourceLoader],
+    amfConfigurationStateDeltas: Option[AMFConfigurationStateManager] = None)
     extends PlatformSecrets {
   private implicit var configuration: AMFConfiguration = initialConfig
 
+  private var innerAmfConfigurationStateDeltas: AMFConfigurationStateManager =
```

Comment:
Maybe a little picky on my part but I don't really like the name of the variable. I would remove the word "Deltas" and let it just be _innerAmfConfigurationState_ or _innerAmfConfigurationStateManager_